htp/htp_private.h

```diff
@@ -233,6 +233,9 @@ htp_status_t htp_tx_res_process_body_data_ex(htp_tx_t *tx, const void *data, siz
 htp_status_t htp_tx_urldecode_uri_inplace(htp_tx_t *tx, bstr *input);
 htp_status_t htp_tx_urldecode_params_inplace(htp_tx_t *tx, bstr *input);
 
+size_t strlcat(char *dst, const char *src, size_t size);
```

Comment:
This should only be defined if #ifndef HAVE_STRLCAT, as some compilers/libc's define their own. (clang on OSX10.10 does)